include/ghex/rma/range_traits.hpp

```diff
@@ -0,0 +1,30 @@
+/* 
+ * GridTools
+ * 
+ * Copyright (c) 2014-2020, ETH Zurich
+ * All rights reserved.
+ * 
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * 
+ */
+#ifndef INCLUDED_GHEX_RMA_RANGE_TRAITS_HPP
+#define INCLUDED_GHEX_RMA_RANGE_TRAITS_HPP
+
+namespace gridtools {
+namespace ghex {
+namespace rma {
+
+template<template<typename> class RangeGen>
```

Comment:
A comment on why traits are needed here?

Reply:
You are correct, we can probably remove the traits class altogether. I will change this.